### ISSUE TITLE
Fix prm section names in benchmark folders

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1638]
`benchmarks/include/adaptive.prm.base` still has this section:
```
subsection Initial temperature 
  set Model name = perturbed box
end
```
and so do several of the other benchmark input files. These got forgotten when we renamed these section names recently and will not work any more right now.

@alarshi is taking care of the one file mentioned above, but we need to test and convert the other ones as well.

**Comment by hlokavarapu** [comment]
Looking into making the conversion of parameter subsections.

**Comment by hlokavarapu** [comment]
After going through the benchmarks, I found that none of the parameter files were out of date other than benchmarks/inclusion/global.prm.base and benchmarks/inclusion/adaptive.prm.base

My guess is that because these files do not end with the extension .prm, they were not updated.

**Comment by gassmoeller** [comment]
Yes that was me. I did only update files ending on .prm. Thinking about it there might be other files. Working on a PR

**Comment by gassmoeller** [comment]
Closed by #1646 and #1633.
